src/qt/rpcconsole.cpp

```diff
@@ -392,6 +392,28 @@ void RPCExecutor::request(const QString &command)
     {
         std::string result;
         std::string executableCommand = command.toStdString() + "\n";
+
+        // Catch the console-only-help command before RPC call is executed and reply with help text as-if a RPC reply.
+        if(executableCommand == "help-console\n")
+        {
+            Q_EMIT reply(RPCConsole::CMD_REPLY, QString(("\n"
+                "This console accepts RPC commands using the standard syntax.\n"
+                "   example:    getblockhash 8\n\n"
+
+                "This console can also accept RPC commands using bracketed syntax.\n"
+                "   example:    getblockhash(8)\n\n"
+
+                "A space or a comma can be used to separate arguments for either syntax.\n"
+                "   example:    sendtoaddress <address> <amount>\n"
+                "               sendtoaddress,<address>,<amount>\n\n"
+
+                "Commands may be nested when specified with the bracketed syntax.\n"
+                "   example:    getblockinfo(getblockhash(0),true).\n\n"
+
+                "Result values can be queried with a non-quoted string in brackets.\n"
+                "   example:    getblock(getblockhash(0) true)[height]\n\n" )));
```

Comment:
maybe add an example that uses access-by-index like `getblock(getblockhash(0) true)[tx][0]`?

Reply:
Agreed, should have included that.

Reply:
Here the use of "bracket" is correct, referring to the `[]`.